docs/oauth1/server.rst

```diff
@@ -2,7 +2,7 @@
 Creating a Provider
 ===================
 
-OAuthLib is a dependency free library that may be used with any web
+OAuthLib is a framework independent library that may be used with any web
```

Comment:
Because OAuthLib has dependencies, so it is not "dependency free". The description of "dependency free" would mean it does not need any other package to run. OAuthLib (as indicated in the _requirements.txt_ file) needs PyJWT, blinker and cryptography packages to function.

I'm trying to guess what the original author really meant, and assume they meant to say it can be used with any framework (e.g. doesn't have to be Django) rather than "dependency free" (which it is clearly not, since it has dependencies).